1/contrib/openshift/configuration/jobs/OpenShift Sample/config.xml

```diff
@@ -57,11 +57,13 @@
       <namespace>test</namespace>
       <testTag>latest</testTag>
       <prodTag>prod</prodTag>
-      <testStream>origin-nodejs-sample</testStream>
-      <prodStream>origin-nodejs-sample</prodStream>
+      <testStream>origin-ruby-sample</testStream>
+      <prodStream>origin-ruby-sample</prodStream>
       <authToken></authToken>
     </com.openshift.jenkins.plugins.pipeline.OpenShiftImageTagger>
 
+    <!--
+    FIXME: There is no such deployment config created by sample-app.
```

Comment:
@gabemontero @bparees ok to remove this step? it will always fail because the sample-app in origin does not have that deployment config.